src/UserFrontend.php

```diff
@@ -61,10 +64,26 @@ public static function login_head() {
    */
   public static function login_form_defaults(array $args) {
     $args['value_remember'] = TRUE;
+    $args['remember'] = FALSE;
     return $args;
   }
 
   /**
+   * @implements login_form
+   */
+  public static function login_form() {
+    ob_start(__CLASS__ . '::removesForgetmenotCheckbox');
```

Comment:
As I already told before in the PR description, reading lots of documentation online, we need two different steps in order to hide the checkbox everywhere. The `remember` parameter hide the checkbox everywhere but not on the `wp-login.php` page. To hide it as well on the `wp-login.php` we need to use the `login_form` action as well.

About the output buffer is needed to alter the `$content` since, without it, the parameter is empty. `login_form` is an action and is called just before the remember me checkbox is output so this is why we need to start an output buffer to alter the content is coming next.  Before decide to adopt this solution, I stutied couple of plugins that remove the checkbox and both of them use the same approach. 

Agree about the buffer end/clean. Where to do it in your opinion? At the end of the `removesForgetmenotCheckbox` function? 

On a separate topic: should, in your opinion, `removesForgetmenotCheckbox` be more consistent if called `removedRememberMeCheckbox`?